src/lxc/pam/meson.build

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+pam_cgfs_includes = include_directories(
+    '.',
+    '../',
+    '../cgroups',
+    '../../include',
+    '../../../')
+
+pam_cgfs_sources = files(
+    'pam_cgfs.c',
+    '../file_utils.c',
+    '../file_utils.h',
+    '../macro.h',
+    '../memory_utils.h',
+    '../string_utils.c',
+    '../string_utils.h') + include_sources
+
+pam_cgfs_common_dependencies = declare_dependency(
+    include_directories: pam_cgfs_includes,
+    sources: pam_cgfs_sources,
+    dependencies: [
+        pkgconfig_libs,
+    ])
+
+pam_cgfs = shared_module(
+    'pam_cgfs',
+    pam_cgfs_sources,
+    dependencies: pam_cgfs_common_dependencies,
+    name_prefix: '',
+    install_mode: 'rw-r--r--', # 644
+    install: want_pam_cgroup,
```

Comment:
You always build this shared module by default, but only install it if it is enabled via an option? This doesn't seem correct... if it is not enabled, then the pam library won't be found and its headers may not be installed, and the shared_module will fail to link.

I think you should just have it always install, but just do not `subdir()` into this directory if the option is disabled (or more generally, if pam is not found).

Reply:
Sounds good!